### ISSUE TITLE
PLOTS offline after v1.3.0 (Build 3) Update

### ISSUE THREAD

**Issue by Saganaki22** [hpool-dev/chia-miner#258]
I updated HPOOL miner to the newer build since seeing the update, and since then ive had 0 income and all my plots are offline in dashboard but miner shows online ?
Any fix for win ?

**Comment by RishabhKumarRK** [comment]
right, I am having the same problem. My plots are also offline but the miner is working fine.

**Comment by Saganaki22** [comment]
> right, I am having the same problem. My plots are also offline but the miner is working fine.

i just got a msg from a friend who had the exact same issue, he said his online dashboard updated about  4 hours after initial update to 1.3.0 (Build 3) with all his plots back online

**Comment by RishabhKumarRK** [comment]
mine are also have come online but will we getting the revenue of this lost time?
cause the miner was just working fine

**Comment by Saganaki22** [comment]
to be honest i don't know, my plots are all offline still for some reason

**Comment by RishabhKumarRK** [comment]
no we will not be getting any lost revenue as i just received block income and it normal as before 
-hpool will make money by any means

**Comment by Saganaki22** [comment]
how long did it take for your plots to come online?

**Comment by RishabhKumarRK** [comment]
i restart it couple of time but took around 4 hrs

**Comment by NikolaySayfuranov** [comment]
The same problem. Part of plots are online, but some of them are offline. It's been already for 10 hours without changing.
Also in GUI miner, if I see properties of plots folder, I see that exactly this plots are unsigned. Other plots has date and time of signiture, but this plots has "--".

**Comment by RishabhKumarRK** [comment]
if problem not solved yet then try removing directory for unsigned plots and adding them back, after that restart it 
that might help.

**Comment by xaviveer** [comment]
Same problem here, no plots appearing and miner "operating" status is --
